src/main/fc/rc_adjustments.c

```diff
@@ -197,6 +198,37 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
     }
 };
 
+#ifdef OSD
+#if (FLASH_SIZE > 2048)
+static adjustmentLabel_t adjustmentLabels[] = {
```

Comment:
Note, you don't use the `id`, so you can change this to an array of strings.

Reply:
done